actionpack/test/controller/render_test.rb

```diff
@@ -560,6 +565,11 @@ def test_head_with_status_code_first
     assert_equal "something", @response.headers["X-Custom-Header"]
     assert_response :forbidden
   end
+
+  def test_head_and_return
+    get :head_and_return
+    assert_equal 200, @response.response_code
```

Comment:
Don't think we need to test that `head` sets the correct response code here. That's tested elsewhere. Instead, how about being explicit that an error will be raised if `head` is not returning a truthy value? Example:

```ruby
def test_head_returns_truthy_value
  assert_nothing_raised do
    get :head_and_return
  end
end
```